### PR TITLE
Lab #2. Alexander Kurmazov

### PR DESCRIPTION
**Pull request by AKurmazov**

### CODE REVIEW

File: app_python/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+CODE := server
```
Comment (afonyaa) [inline app_python/Makefile:1]: 🥇 🥇 🥇

**Review by afonyaa — APPROVED** [review]
Good job! Nothing to find fault

**Comment by none-word** [comment]
Good job!

**Comment by Maxkoz777** [comment]
Usage of a Makefile is indeed a good practice, well done

**Comment by Homa3030** [comment]
:+1:

**Pull request closed without merging**